Description: this fixes the build failure and need for internet access
Author: Gianfranco Costamagna <locutusofborg@debian.org>

--- /dev/null
+++ haskell-iso8601-time-0.1.4/Setup.hs
@@ -0,0 +1,2 @@
+import Distribution.Simple
+main = defaultMain
